src/Infiltration/ui/GameTimer.tsx

```diff
@@ -14,7 +15,7 @@ export function GameTimer({ endTimestamp }: GameTimerProps): React.ReactElement
   // actually begin it until later. We're using a ref to keep this updated in
   // a very low-overhead way.
   const state = changeRef.current;
-  if (state?.endTimestamp !== endTimestamp) {
+  if (state.endTimestamp !== endTimestamp) {
```

Comment:
Weird, I know TS was complaining about null at some point. (But it shouldn't be.)